src/storage/SimpleResourceStore.ts

```diff
@@ -27,7 +27,7 @@ export class SimpleResourceStore implements ResourceStore {
       '': {
         dataType: DATA_TYPE_BINARY,
         data: streamifyArray([]),
-        metadata: { raw: [], profiles: []},
+        metadata: { raw: [], profiles: [], contentType: 'text/turtle' },
```

Comment:
Just for my information, is that technically a conflict with `DATA_TYPE_BINARY`?

Reply:
No that's actually correct. We have 2 data types currently: binary and quads, so since turtle is a string stream it would be binary. Tbh, something could be said about just removing the dataType field and simply looking at the content-type, since currently everything that is not `internal/quads` would have the data type binary. Could also extend to more datatypes (string I guess?), but don't really see where we would make use of that specifically.

Reply:
Got it, the way it is now makes sense.